### DISCUSSION TITLE
port to embree4

### DISCUSSION THREAD

**Discussion by githubkusi** [microsoft/vcpkg#38935] [General]
Hi @zhiwei-c 

You started a [pull request](https://github.com/microsoft/vcpkg/pull/38179) of a new port for embree4. Thanks for your effort! The PRQ was closed, why so? Do you intend to continue the new port?

**Comment by autoantwort** [comment]
Because it added a new port embree4 instead of updating to existing one to version 4 iiuc.

**Reply by zhiwei-c** [reply]
Yes.

**Reply by githubkusi** [reply]
but this is actually wanted to have both embree3 and embree4 available in parallel, isn't it? Some libs depends on v3, others on v4

**Comment by zhiwei-c** [comment]
Now the libigl library strictly depends on embree3 if it enables feature "embree". Thus, the old port cannot be edit directly.
It is better to use your own port if embree4 needs to be installed.  https://learn.microsoft.com/en-us/vcpkg/concepts/overlay-ports
